integration-tests/gatsby-cli/__tests__/new.js

```diff
@@ -78,4 +78,10 @@ describe(`gatsby new`, () => {
 
     expect(code).toBe(1)
   })
+
+  it(`an empty gatsby new (without args) runs create-gatsby`, () => {
```

Comment:
Very very nitpicky: the test name is not a "readable" sentance. The "it" at the begining aims to start a sentance, something like `it runs create-gatsby when no arguments are provided to gatsby new`

Reply:
I love your nitpicks, thanks Marvin 🙂